experiments/extract_all_data.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# The below must match the path to the bin/ directory of the DreamCoder
+# PLDI artifact, and must not include a trailing /
+ARTIFACT_BIN_PATH="/scratch/theoxo/artifact/bin"
```

Comment:
maybe have this script take "/scratch/theoxo/artifact" as a commandline arg and it calculates these two from that? so it works with other peoples stuff

Reply:
Nice small change, added in [8d01421](https://github.com/mlb2251/stitch/pull/87/commits/8d014211315a147353c291495e133bbb1dfdd7ad).